x/lockup/keeper/lock_test.go

```diff
@@ -534,6 +535,87 @@ func (s *KeeperTestSuite) TestCreateLock() {
 	s.Require().Equal(sdk.NewInt(30), balance.Amount)
 }
 
+func (s *KeeperTestSuite) TestSetLockRewardReceiverAddress() {
+	testCases := []struct {
+		name                  string
+		isnotOwner            bool
+		lockID                uint64
+		useNewReceiverAddress bool
+		expectedError         bool
```

Comment:
nit: test errors

Reply:
Addressed in https://github.com/osmosis-labs/osmosis/pull/5281/commits/80b11f5b33ad87b407ed876e98ece45f4bfa6191